bazel/swift_static_framework.bzl

```diff
@@ -142,6 +142,7 @@ def swift_static_framework(
         name,
         module_name = None,
         srcs = [],
+        resources = [],
```

Comment:
I think the builtin objc framework uses `data` for this and imo we should follow that example.